bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/chlorinator.xml

```diff
@@ -67,7 +66,7 @@
 	</channel-type>
 
 	<channel-type id="chlorError">
-		<item-type>Number</item-type>
+		<item-type>String</item-type>
```

Comment:
As the item type is updated, you probably have to provide thing upgrade instructions for all channels of this type.
I say "probably" because I am not 100% sure. @jlaur: can you confirm ?

Same for channel type `status` below

Or are these channels all built dynamically ?

Reply:
These channels are not built dynamically.  I added an alert in PR #[1608](https://github.com/openhab/openhab-distro/pull/1608)

Reply:
Thing upgrade instructions should be provided. Documentation can be found here:
https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

Reply:
Added.  Can you let me know what needs done with the alert (i.e. delete, how to update version)?